fluent-bit/fluent-bit-go-loki/Makefile

```diff
@@ -0,0 +1,5 @@
+fluent-bit:
```

Comment:
If you want to move this into our Makefile later, let's create an issue and we will get back at it.

Reply:
I've imported @sh0rez patch:
https://github.com/grafana/loki/pull/847/commits/16af27376b8a2fe190b3f507fc59cfca78a9449f